src/materials/Material.js

```diff
@@ -294,6 +303,29 @@ class Material extends EventDispatcher {
 		data.stencilZFail = this.stencilZFail;
 		data.stencilZPass = this.stencilZPass;
 
+		if ( this.clippingPlanes !== null ) data.clippingPlanes = this.clippingPlanes;
```

Comment:
This property is an array of `THREE.Plane`. You can serialize it like that.

Reply:
@Mugen87  Already fixed. Thanks. 👍 
![image](https://user-images.githubusercontent.com/19889378/122920203-b7121180-d393-11eb-8e32-d5929b14143f.png)